transcom/mymove
# INT B-20083

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20083)

## Summary

This branch fixes 2 issues:
1. The Prime Simulator would not update the "Actual" weight values for Pro Gear or Spouse Pro Gear when the "Update Shipment" button was used.
2. On the "Review Weights" page during PPM Closeout, for partial shipments (HHG + PPM), the actual pro gear weights were not showing up for the HHG shipment.

To address this:
1. The prime.yaml file, updateMTOShipment service, and endpoint handler have been edited to include definitions for "actualProGear" and "actualSpouseProGear".
2. The standalone "Pro Gear" row in the Review Weights page has been removed, and the pro gear values now appear as columns in the HHG shipment's row.

### How to test

1. As a customer, create a move that contains both an HHG and a PPM shipment
2. As an SC, approve the move the usual way
3. As customer, upload PPM docs
4. As TOO, navigate to the move and approve the move for counseling.
5. As the Prime, navigate to the move, find the HHG shipment and click "Update Shipment". Here, update both "Actual" pro gear weight fields. (If you get an error saying the time to edit weights has passed, put the dates at the top of the form further into the future).
6. As the SC, navigate to the move and click "Review Shipment Weights". At the bottom of the screen, your HHG shipment should look similar to the screenshot below, but displaying whichever numbers you input as the Prime.

![Screenshot 2024-05-29 at 2 45 16 PM](https://github.com/transcom/mymove/assets/146959895/8cfd9fb3-fb78-4183-a988-bcd73e5672c4)
